src/Components/Topic.jsx

```diff
@@ -60,6 +66,12 @@ function Topic() {
                     <Card.Text className="articleText">
                    By {article.author}
                 </Card.Text>
+                <Card.Text className="articleText">
```

Comment:
rather than having an if/else here and checking if it's the first time and passing 'none' - you could set the states of sort_by and order to default values that actually exist, so that the articles and the api call still work. then if either of those states are changed, the useeffect can run then the api call will be made with updated sort by/order

in that case we wouldn't need an if/else, it would just always run a similar way